src/internal/utils/bucketHelper.ts

```diff
@@ -456,6 +468,10 @@ export function getAllAttributeItems(buckets: IBucket[]): IBucketItem[] {
     return getAllItemsByType(buckets, [ATTRIBUTE, DATE]);
 }
 
+export function getAllMeasureItems(buckets: IBucket[]): IBucketItem[] {
```

Comment:
A unit test is missing for this utility.

Reply:
I've removed the `export` keyword so now it's tested only as part of filters sanitization